hiroaki-core/src/main/java/me/jorgecastillo/hiroaki/dispatcher/DispatcherRetainer.kt

```diff
@@ -2,7 +2,7 @@ package me.jorgecastillo.hiroaki.dispatcher
 
 import java.io.File
 
-internal object DispatcherRetainer : Retainer {
```

Comment:
I cannot keep this `internal` since it is now used in the other module. One option could be to move it into `internal` package name.

Reply:
How would that restrict visibility over it? linting? It's fine to remove internal. Not ideal but Kotlin lacks a protected multi module modifier.

Reply:
We could move it to a package with internal name in it. Not sure if there is any other way.

Reply:
I'm not sure I follow, by package you mean actual package or you trying to refer to Gradle module? How moving it to an `internal` package would restrict visibility?

Reply:
It wouldn't restrict programmatically but it is a common practice. When a class is in a package named internal, it is an indication that the user takes the responsibility. And library author would be more free to change internal classes.